gdrivefs/conf.py

```diff
@@ -54,5 +57,7 @@ def get(key):
     def set(key, value):
         if key not in Conf.__dict__:
             raise KeyError(key)
+        elif key is "delete_to_trash":
```

Comment:
In get() make sure to use an inequality, not is. is is only used when checking that one instance is the same as another. By contrast, creating two different objects (instances) from the same class should always return False. Note that string caching may create inconsistent results when comparing strings. Consequently, you should only use is for comparing False, True, and None.